packages/client/tests/unit.utils.test.ts

```diff
@@ -127,49 +127,4 @@ describe('buildFileTree()', () => {
       normalizeWindowsPaths(ignoreList).sort()
     );
   });
-
-  it('should find root files but ignore all .output files when prebuilt=false and rootDirectory=root', async () => {
```

Comment:
I think we need to keep a test with `rootDirectory` assigned, to make sure this change works as expected.

Reply:
Oh I see, this was a unit test so the prop was removed 😆